core/backend/src/SchemaSync.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module SQLiteDb
+ */
+
+import { CloudSqlite } from "./CloudSqlite";
+import { VersionedSqliteDb } from "./SQLiteDb";
+
+
+/** @beta */
+export namespace SchemaSync {
+
+  /**
+   * A SQLite database for storing schemas.
+   */
+  export class SchemaSyncDb extends VersionedSqliteDb {
+    public override readonly myVersion = "1.0.0";
+    protected override createDDL() { }
+  }
+
+  const defaultDbName = "SchemaSyncDb" as const;
+
+  /**
+   * Provides access to a cloud-based `SharedSchemaChannelDb` to hold ECSchemas.
+   * `SchemaSync.ChannelDb`s that are stored in cloud containers require an access token that grants permission to read and/or write them.
```

Comment:
you can remove the comments copied from CloudSqlite.DbAccess

Reply:
fixed